cmd/snap-confine/ns-support.h

```diff
@@ -53,9 +53,11 @@ void sc_reassociate_with_pid1_mount_ns(void);
  * where namespaces are kept (/run/snapd/ns) is correctly prepared as described
  * above.
  *
+ * Optional experimental features can be enabled via arguments.
+ *
  * For more details see namespaces(7).
  **/
-void sc_initialize_mount_ns(void);
+void sc_initialize_mount_ns(bool experimental_features);
```

Comment:
Mild -1 on this part, just call it what it is. Bonus tip for changing flags to `n<<1` style so that this actually checks if a flag is on or off.

Reply:
https://github.com/snapcore/snapd/pull/7389

Reply:
Thanks! Reviewed